### ISSUE TITLE
Godot 4.0rc1 broke this

### ISSUE THREAD

**Issue by elenakrittik** [KoBeWi/Godot-Universal-Fade#2]
After rc1 i started getting these errors:
```
E 0:00:01:0848   Fade.gd:88 @ _fade(): Failed setting key 'opacity' at time 0 in Animation 'FadeOut' at Node '/root/Fade/AnimationPlayer', Track 'TextureRect:material:opacity'. Check if the property exists or the type of key is right for the property.
  <C++ Source>   scene/animation/animation_player.cpp:1171 @ _animation_update_transforms()
  <Stack Trace>  Fade.gd:88 @ _fade()
                 Fade.gd:13 @ fade_out()
                 launch_screen.gd:14 @ _process()
```
and multiple
```
E 0:00:01:0852   _animation_update_transforms: Failed setting key 'opacity' at time 0.006537 in Animation 'FadeOut' at Node '/root/Fade/AnimationPlayer', Track 'TextureRect:material:opacity'. Check if the property exists or the type of key is right for the property.
  <C++ Source>   scene/animation/animation_player.cpp:1171 @ _animation_update_transforms()
```
after it.
There is no longer transitions between completely faded out screen and completely faded in screen. Tried changing property path in various ways but none seem to work. No idea what PR caused this issue

**Comment by elenakrittik** [comment]
https://user-images.githubusercontent.com/77104725/218134785-45a9f8b4-63dd-4fc1-a1bc-b32c6878fd95.mp4

**Comment by KoBeWi** [comment]
Thanks for the report, fixed in ddab6c26cf174fab107fbd6b4e66a5a350e83c22

**Comment by elenakrittik** [comment]
Thanks!
